gammapy/background/energy_offset_array.py

```diff
@@ -3,12 +3,35 @@
 import numpy as np
 from astropy.coordinates import Angle
 from astropy.units import Quantity
+from astropy.table import Table, Column
+from astropy.io import fits
+from ..utils.fits import table_to_fits_table
 
 __all__ = [
     'EnergyOffsetArray',
 ]
 
 
+def _make_bin_edges_array(lo, hi):
```

Comment:
Instead of duplicating this utility function, can you please remove it here and import it from the other file?
```
from .cube import _make_bin_edges_array
```